packages/react-conformance/src/defaultTests.tsx

```diff
@@ -0,0 +1,69 @@
+import { TestObject, TestingOptions } from './types';
+import { ComponentDoc } from 'react-docgen-typescript';
+import { mount } from 'enzyme';
+import * as ReactIs from 'react-is';
+import * as React from 'react';
+
+/** For a reference of the tests, check the file rules.md in @fluentui/react-conformance */
+
+export const defaultTests: TestObject = {
+  /** Component has a docblock with 5 to 25 words */
+  'v0-1a': (componentInfo: ComponentDoc, componentPath: string, testInfo: TestingOptions) => {
```

Comment:
We should probably come up with more descriptive names--these will be meaningless except for anyone looking at our spec.

Reply:
I added a test.md file with all the rules that we are going to add. I also added a comment at the top saying where to find it but I agree, I just wasn't really sure what to call them.